manifests/k/KuaiFan/DooTask/0.44.3/KuaiFan.DooTask.installer.yaml

```
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: KuaiFan.DooTask
+PackageVersion: 0.44.3
+InstallerType: nullsoft
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+ProductCode: 9a445616-2838-509d-bd29-65c401183c4f
+ReleaseDate: 2025-03-09
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/kuaifan/dootask/releases/download/v0.44.3/DooTask-v0.44.3-win-x64.exe
+  InstallerSha256: 43D87FF7DD5168AD34D722C609C8150BB3EDEB9270C4FE122B9AD27B96A2B653

```

```suggestion
  InstallerSha256: 70B7C083DC47BBBE7C0FB01108EE6901B5F85D421A37469B0E905D61B417CA3C
```

(Automated response - build 994.)